vm/vm.go

```diff
@@ -437,7 +437,7 @@ func (mon *monitor) createReport(defaultError string) *report.Report {
 }
 
 func (mon *monitor) waitForOutput() {
-	timer := time.NewTimer(waitForOutputTimeout * mon.inst.pool.timeouts.Scale)
+	timer := time.NewTimer(vmimpl.WaitForOutputTimeout)
```

Comment:
The `* mon.inst.pool.timeouts.Scale` part is lost?

Reply:
Yes. It's somewhat tricky to scale it inside `vmimpl.Multiplex()` + I'm not sure we need it that much. A 10 seconds delay sounds reasonable even for emulated VMs.

Reply:
Added a commit to pass slowdown to `Multiplex()`.